cyclonedx/output/serializer/json.py

```diff
@@ -59,6 +59,10 @@ def default(self, o: Any) -> Any:
         if isinstance(o, XsUri):
             return str(o)
 
+        # PackageURL
```

Comment:
Interesting you've added this code @Kravcu - myself and the other maintainers struggled with JSON serialisation of `PackageURL` objects as they are extensions of `tuple`, which the JSON serializer handles before the `default()` method is called - so this code will never be reached.

Our workaround was lines 86-89 below.